dist/adguard_companies.json

```diff
@@ -0,0 +1,60 @@
+{
+    "timeUpdated": "2022-04-13T08:24:53.623Z",
+    "companies": {
```

Comment:
Here I've collected companies used in adguard.json.
Although this list should be reviewed.